afs/media/js/views/components/workflows/create-project-workflow/project-name-step.js

```diff
@@ -99,6 +99,12 @@ define([
                 return;
             }
     
+            if (!self.typeValue()) {
```

Comment:
I see this follows the same pattern as the `Project Name` node, but we should actually handle for missing required values using the response from the server.  That way the error message will be properly localized. We can address this for both nodes in another PR. This fix prevents the step from breaking, though, so it's a good stop-gap.